### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by LiangZhou-CTY**
Signed-off-by: Liang Zhou <zhoul110@chinatelecom.cn>

### CODE REVIEW

File: libcontainer/README.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ state, err := container.State()
 #### Checkpoint & Restore
 
 libcontainer now integrates [CRIU](http://criu.org/) for checkpointing and restoring containers.
-This let's you save the state of a process running inside a container to disk, and then restore
+This let you save the state of a process running inside a container to disk, and then restore
```
Comment (thaJeztah) [inline libcontainer/README.md:320]: This doesn't look correct; I think the correct fix here would be
```suggestion
This lets you save the state of a process running inside a container to disk, and then restore
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: libcontainer/README.md (hunk 1 above)
Comment (LiangZhou-CTY) [reply libcontainer/README.md]: Thanks for the correction, I modified it.

**Review by thaJeztah — APPROVED** [review]
Thanks!

LGTM (not a maintainer)

**Review by kolyshkin — COMMENTED** [review]
LGTM

**Review by kolyshkin — APPROVED** [review]

**Review by mrunalp — APPROVED** [review]

**Pull request merged**